rpc-server/src/modules/queries/contract_runner/code_storage.rs

```diff
@@ -0,0 +1,305 @@
+use std::collections::HashMap;
+
+use crate::modules::queries::utils::get_state_key_value_from_db;
+use database::ReaderDbManager;
+use futures::executor::block_on;
+
+pub type Result<T> = ::std::result::Result<T, near_vm_runner::logic::VMLogicError>;
+
+pub struct CodeStorage {
+    db_manager: std::sync::Arc<Box<dyn ReaderDbManager + Sync + Send + 'static>>,
+    account_id: near_primitives::types::AccountId,
+    block_height: near_primitives::types::BlockHeight,
+    validators: HashMap<near_primitives::types::AccountId, near_primitives::types::Balance>,
+    data_count: u64,
+    is_optimistic: bool,
+    prefetched_data:
```

Comment:
This is already deleted in https://github.com/near/read-rpc/pull/259. Observations have shown that it is not efficient to request the whole stack. Therefore, this functionality was removed.